docs/teams/android/project_setup/00_env_setup.md

```diff
@@ -0,0 +1,167 @@
+# Local Environment Setup
+
+This guide describes how to set up your development environment for Kotlin Multiplatform (KMP) apps and how to run KMP based app on virtual and physical android device. It applies to both Android and iOS developers.
+
+## Prerequisites
+
+- macOS (required to compile iOS)
+- JDK 17+
+- Android Studio as the Kotlin + Android IDE
+- Xcode as the Swift IDE
+- (Optional) JetBrains Fleet for IDE users who prefer it
```

Comment:
KMP with Fleet [is dead](https://blog.jetbrains.com/kotlin/2025/02/kotlin-multiplatform-tooling-shifting-gears/)